### PR TITLE
Cake 5 Update

### PR DESCRIPTION
**Pull request by idfake**
As the title suggests.
I made minimal changed to provide Cake5 compat.

### CODE REVIEW

File: src/Excel/Manager.php [hunk 1]
```diff
@@ -354,14 +354,14 @@ public function getReader(File $file, array $options = [])
     public function getWriter(Spreadsheet $excel, File $file, array $options = [])
     {
         if (!$file->exists()) {
```
Comment (robertpustulka) [inline src/Excel/Manager.php:356]: Should this be renamed to getSize() as well?

**Comment by robertpustulka** [comment]
@idfake please reopen targeting master branch

**Comment by idfake** [comment]
> @idfake please reopen targeting master branch
sure. #20

File: src/Excel/Manager.php (hunk 1 above)
Comment (idfake) [reply src/Excel/Manager.php:356]: yeah, missed that.

**Pull request closed without merging**